modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/suspended/SuspendToKindSyntax.kt

```diff
@@ -0,0 +1,8 @@
+package arrow.effects.typeclasses.suspended
+
+import arrow.typeclasses.suspended.BindSyntax
+
+interface SuspendToKindSyntax<F> : BindSyntax<F> {
+
+
```

Comment:
??? TODO or something

Reply:
addressed in https://github.com/arrow-kt/arrow/pull/1263/commits/abde3ac466da7379d06e77dea1986c89ee69b57d